Hymns/Tags/TagEntity.swift

```diff
@@ -27,3 +33,7 @@ class TagEntity: Object, Identifiable {
         return ("\(hymnIdentifier.hymnType):\(hymnIdentifier.hymnNumber):\(hymnIdentifier.queryParams ?? [String: String]()):\(tag)")
     }
 }
+
+enum TagColor: Int {
+        case blue, green, yellow, red
```

Comment:
I think you should have a default case also

Reply:
I could not just add default since it is an enum keyword that doesn't work here but only in switch statements. I added a none as seen here https://useyourloaf.com/blog/enum-raw-values-and-failable-initializers/
Does that still accomplish what you are after?

Reply:
yup, that works!

Reply:
hmmm, actually, what if we made TagColor an optional value? I'm not sure if that would be better or worse or would make no difference. What do you think would be the pros and cons of that versus creating a special "none" enum value?

Reply:
I think it would make no difference. If you want I can do it though. I don't think it would make a difference since we will always have a tagColor stored. I think the only pro is it may look slightly better in code. But the con is we have to deal with an optional, unwrap it etc.. 

Let me know what you think

Reply:
ok. It's probably simpler then to just keep the "none" enum value, since we can just use a switch statement instead of dealing with wrapping/unwrapping optionals!